.github/workflows/unit-test-shards.json

```diff
@@ -1,14 +1,62 @@
-  "openedx-2": "openedx/core/djangoapps/geoinfo/ openedx/core/djangoapps/header_control/ openedx/core/djangoapps/heartbeat/ openedx/core/djangoapps/lang_pref/ openedx/core/djangoapps/models/ openedx/core/djangoapps/monkey_patch/ openedx/core/djangoapps/oauth_dispatch/ openedx/core/djangoapps/olx_rest_api/ openedx/core/djangoapps/password_policy/ openedx/core/djangoapps/plugin_api/ openedx/core/djangoapps/plugins/ openedx/core/djangoapps/profile_images/ openedx/core/djangoapps/programs/ openedx/core/djangoapps/safe_sessions/ openedx/core/djangoapps/schedules/ openedx/core/djangoapps/self_paced/ openedx/core/djangoapps/service_status/ openedx/core/djangoapps/session_inactivity_timeout/ openedx/core/djangoapps/signals/ openedx/core/djangoapps/site_configuration/ openedx/core/djangoapps/system_wide_roles/ openedx/core/djangoapps/theming/ openedx/core/djangoapps/user_api/ openedx/core/djangoapps/user_authn/ openedx/core/djangoapps/util/ openedx/core/djangoapps/verified_track_content/ openedx/core/djangoapps/video_config/ openedx/core/djangoapps/video_pipeline/ openedx/core/djangoapps/waffle_utils/ openedx/core/djangoapps/xblock/ openedx/core/djangoapps/xmodule_django/ openedx/core/djangoapps/zendesk_proxy/ openedx/core/djangolib/ openedx/core/lib/ openedx/core/tests/ openedx/core/tests/ openedx/features/ openedx/testing/ openedx/tests/",
-  "cms-1": "cms/djangoapps/api/ cms/djangoapps/cms_user_tasks/ cms/djangoapps/course_creators/ cms/djangoapps/export_course_metadata/ cms/djangoapps/maintenance/ cms/djangoapps/models/ cms/djangoapps/pipeline_js/ cms/djangoapps/xblock_config/ cms/envs/ cms/lib/",
-  "cms-2": "cms/djangoapps/contentstore/",
-  "common-1": "common/djangoapps/",
-  "common-2": "common/lib/"
+  "lms-1": {
+    "path": ["lms/djangoapps/badges/","lms/djangoapps/branding/","lms/djangoapps/bulk_email/","lms/djangoapps/bulk_enroll/","lms/djangoapps/bulk_user_retirement/","lms/djangoapps/ccx/","lms/djangoapps/certificates/","lms/djangoapps/commerce/"],
```

Comment:
Please format this list with one item per line so that diffs will make more sense.  (Also, "path" is an odd name for a list of paths?)

Reply:
current formatting is more readable since it shows which apps are being tested against `cms` or `lms` settings, in my opinion, this is easier to understand for developers not aware of the whole sharding strategy
we can change the formatting but we'd have to scroll to the end to see the config for each shard

Reply:
Sorry, I worded it poorly.  I was suggesting it look like this:
```
{
  "lms-1": {
    "settings": "lms.envs.test",
    "paths": [
      "lms/djangoapps/badges/",
      "lms/djangoapps/branding/",
      "lms/djangoapps/bulk_email/",
      "lms/djangoapps/bulk_enroll/",
      ..etc..
    ]
  },
  "lms-2": {
    ...
  },
  ...
}
```